orc8r/gateway/python/magma/ctraced/command_builder.py

```diff
@@ -87,6 +72,9 @@ def build_trace_command(
             for interface in interfaces:
                 command.extend(["-i", interface])
 
+        if len(capture_filters) > 0:
```

Comment:
Let's add some defensive checks here to ensure tshark filters is only passed here. We shouldn't let a lower priority user pass some arbitrary string who can for e.g. reboot the node.